src/API/OnboardingPlugins.php

```diff
@@ -156,6 +157,19 @@ public function update_item_permissions_check( $request ) {
 		return true;
 	}
 
+	/**
+	 * Create an alert notification in response to an error installing a plugin.
+	 *
+	 * @param string $slug The slug of the plugin being installed.
+	 */
+	private function create_install_plugin_error_inbox_notification_for_jetpack_installs( $slug ) {
+		if ( 'jetpack' !== $slug ) {
```

Comment:
Do we also want to check for `woocommerce-services` here?

Additionally, I think it might be nice to move some of this logic inside the `possibly_add_install_jetpack_note` method and check there instead.

Reply:
It's also likely that we'll be making this plugins API less onboarding specific and more for general use.

Since we don't know exactly how that's going to flesh out yet, I think a comment in the above docblock that this should be moved to a filter should be enough.

Reply:
> Do we also want to check for `woocommerce-services` here?

Sure, but I couldn't actually find where `woocommerce-services` was being installed here, I assumed that they were part of the same extension.

Reply:
> Additionally, I think it might be nice to move some of this logic inside the `possibly_add_install_jetpack_note` method and check there instead.

I'm not sure I agree with this 100% as it means `possibly_add_install_jetpack_note` has to know about slugs, shifting that logic down another making it harder to find, but I'll do this anyway.

Reply:
> I think a comment in the above docblock that this should be moved to a filter should be enough.
Done